po/de.po

```diff
@@ -17849,7 +17880,7 @@ msgid_plural ""
 "%d images are assigned this tag!"
 msgstr[0] ""
 "Soll das Tag „%s“ wirklich gelöscht werden?\n"
-"%d Bild ist dieses Tag zugewiesen."
+"%d Bild ist dieses Tag zugewiesen!"
```

Comment:
Why the exclamation point? This is not needed and inconsistent with the plural form.